### PR TITLE
Add 'paused' to a list of available statuses in ST2 CLI

### PR DESCRIPTION
**Pull request by benmcbenben**
Add "paused" to possible execution status.

### CODE REVIEW

File: st2client/st2client/commands/action.py [hunk 1]
```diff
@@ -1022,7 +1022,7 @@ def __init__(self, description, app, subparsers, parent_parser=None):
                                                            add_help=True)
 
 
-POSSIBLE_ACTION_STATUS_VALUES = ('succeeded', 'running', 'scheduled', 'failed', 'canceling',
+POSSIBLE_ACTION_STATUS_VALUES = ('succeeded', 'running', 'scheduled', 'paused', 'failed', 'canceling',
```
Comment (armab) [inline st2client/st2client/commands/action.py]: This needs to be fixed too, minor linting issue: 
```
action.py:1025:101: E501 line too long (102 > 100 characters)
```
from https://travis-ci.org/StackStorm/st2/jobs/354356559#L673

**Comment by benmcbenben** [comment]
Changelog updated!

**Comment by armab** [comment]
Thanks for updating changelog :+1: 

How about fixing beforementioned pylint warning? https://github.com/StackStorm/st2/pull/4037#discussion_r175122145
TravisCI is not happy

File: CHANGELOG.rst [hunk 2]
```diff
@@ -67,6 +67,10 @@ Fixed
   to not work. (bugfix) #4001
 
   Contributed by Nick Maludy (Encore Technologies).
+  
+* Fixed missing "paused" status option from "st2 execution list" help output.
```
Comment (armab) [inline CHANGELOG.rst]: Almost there, just add `(bugfix) #4037` at the end of line.

**Comment by benmcbenben** [comment]
Hopefully we're all good now :)

**Pull request merged**